.gitignore

```diff
@@ -111,3 +111,7 @@ python/share/*
 
 python/pybind11/
 
+# Clion project files and directories
+\.idea
+build\.*
```

Comment:
We have directories call build. Put these in your global .gitignore file, not in ngraph. I can help you setup global gitignore.

Reply:
This is "build.*" directory. Anyway, I will add to the global gitignore.